consistency-tests/__tests__/clickables.test.tsx

```diff
@@ -10,7 +10,7 @@ import {render, screen} from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 
 import plus from "@phosphor-icons/core/regular/plus.svg";
-import {ActionItem, OptionItem} from "@khanacademy/wonder-blocks-dropdown";
+import {OptionItem} from "@khanacademy/wonder-blocks-dropdown";
```

Comment:
**note:** Removed `ActionItem` from here as we now delegate the ActionItem's clickable behavior/rendering to `CompactCell`.